app/templates/account.hbs

```diff
@@ -3,19 +3,17 @@
 <header>
   {{#unless config.enterprise}}
```

Comment:
Unrelated, but asking b/c I noticed it in this diff. Can we now remove the `{{unless config.enterprise}}` here?  It doesn't look like `config.billingEndpoint` is configured for enterprise (see [here](https://github.com/travis-ci/travis-web/blob/master/config/environment.js#L66)) , so we could remove it altogether.